### PR TITLE
Create Ticket model

### PR DESCRIPTION
**Pull request by SebastianGarciaMoller**
* Ticket model created
* Added Ticket factory

### CODE REVIEW

File: app/models/ticket.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Ticket < ApplicationRecord
+  belongs_to :list, class_name: 'List'
```
Comment (matias-sclavi) [inline app/models/ticket.rb:4]:
```suggestion
  belongs_to :list
```

File: app/models/ticket.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+  validates :title, presence: true
```
Comment (matias-sclavi) [inline app/models/ticket.rb:6]:
```suggestion
  validates :title, presence: true
  validates :position, presence: true
```

File: db/migrate/20210420210458_create_tickets.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateTickets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tickets do |t|
+      t.string :title
```
Comment (matias-sclavi) [inline db/migrate/20210420210458_create_tickets.rb:4]:
```suggestion
      t.string :title, null: false
```

File: db/migrate/20210420210458_create_tickets.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+      t.string :description
+      t.integer :position
```
Comment (matias-sclavi) [inline db/migrate/20210420210458_create_tickets.rb:6]:
```suggestion
      t.integer :position, null: false, default: 0
```

File: spec/models/ticket_spec.rb [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  it 'creates a valid titcket' do
+    list = FactoryBot.create(:list)
+    ticket = described_class.new(title: 'Test title', list: list)
+
+    is_valid = ticket.save
+
+    expect(is_valid).to be true
+  end
+
+  describe 'validations' do
```
Comment (matias-sclavi) [inline spec/models/ticket_spec.rb:15]: Should we add the `position` validation?

**Review by matias-sclavi — CHANGES_REQUESTED** [review]

File: app/models/ticket.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Ticket < ApplicationRecord
+  belongs_to :list
+
+  validates :title, presence: true
+  validates :position, numericality: { only_integer: true }, presence: true
```
Comment (faqndo97) [inline app/models/ticket.rb:7]: I think presence it's not needed here

File: db/migrate/20210420210458_create_tickets.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+class CreateTickets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tickets do |t|
+      t.string :title, null: false
+      t.string :description
+      t.integer :position, null: false, default: 0
+      t.references :list, foreign_key: { to_table: 'lists' }
```
Comment (faqndo97) [inline db/migrate/20210420210458_create_tickets.rb:7]:
```suggestion
      t.references :list
```

**Review by faqndo97 — CHANGES_REQUESTED** [review]

**Review by faqndo97 — APPROVED** [review]

**Pull request merged**